api/MediaDevices.json

```diff
@@ -402,6 +402,102 @@
             "deprecated": false
           }
         },
+        "constraints_audio_parameter": {
```

Comment:
I'd like to use this data for automation, is there a written rule to name this? What happens when the spec changes the argument name? Will the name here also change?

Reply:
The rules are in https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#parameters-and-parameter-object-features. For the most part the names match the spec and I've been changing some to match, but I can't imagine they all match.

I would guess that in the vast majority of cases there's just one dictionary argument and the argument name can be ignored / assumed to match. That only leaves cases with multiple dictionary arguments needing to match on the name.

Reply:
I've removed these subfeatures since their data matches their parent, and as indicated, there's no need for the individual `audio` and `video` features anyways!